src/components/Button/Button.tsx

```diff
@@ -36,8 +36,7 @@ export interface ButtonProps
   href?: string;
   /** Type of button */
   type?: typeType;
-  /** Value associated to the button */
-  value?: string;
```

Comment:
are we sure we want to remove value as a prop? I think it will be important no?

Reply:
It can be re-added but the children prop overrides it.